src/atldld/utils.py

```diff
@@ -126,8 +131,15 @@ def get_image(image_id, folder=None, expression=False):
         image_url = (
             f"https://api.brain-map.org/api/v2/section_image_download/{image_id}"
         )
+        options = []
+
         if expression:
-            image_url += "?view=expression"
+            options.append("view=expression")
+
+        if downsample:
+            options.append(f"downsample={downsample}")
+
+        image_url += "?" + "$".join(options)
```

Comment:
Is it not `"&"` instead of `"$"` in `join`?

Reply:
Great catch! Thank you!

Reply:
Fixed in f53237d853eee51788a313665e01a6a10373a203